src/modules/count.js

```diff
@@ -0,0 +1,8 @@
+const itemCounter = (item) => item.length;
+
+const displayItemCounted = (shows) => {
+  const counter = document.querySelector('.item-counter');
+  counter.innerText = `TV Shows(${itemCounter(shows)})`;
+};
+
```

Comment:
According to the requirements, the counter should be implemented by counting the actual number of items present in the DOM. To achieve this, you can use the `querySelectorAll` method which would return an array of all specified items.Kindly take this into consideration as it is a core requirement for this milestone